rdrf/rdrf/patient_view.py

```diff
@@ -505,23 +505,27 @@ def get(self, request, registry_code, patient_id):
             login_url = reverse('two_factor:login')
             return redirect("%s?next=%s" % (login_url, patient_edit_url))
 
-        # Section variable hardcoded for now
-        section_blacklist=('Registry', 'Next of Kin')
+
+        registry_model = Registry.objects.get(code=registry_code)
+
+        if "section_blacklist" in registry_model.metadata:
+            section_blacklist = registry_model.metadata["section_blacklist"]
+            section_blacklist = [_("{}".format(x)) for x in section_blacklist]
```

Comment:
section_blacklist = [_(x) for x in section_blacklist]
or 

section_blacklist = map(_,section_blacklist)

Reply:
Fixed.